source/change-streams/tests/README.rst

```diff
@@ -160,6 +160,8 @@ The following tests have not yet been automated, but MUST still be tested
 #. Ensure that a cursor returned from an aggregate command with a cursor id and an initial empty batch is not closed on the driver side.
 #. The ``killCursors`` command sent during the "Resume Process" must not be allowed to throw an exception.
 #. ``$changeStream`` stage for ``ChangeStream`` against a server ``>=4.0`` and ``<4.0.7`` that has not received any results yet MUST include a ``startAtOperationTime`` option when resuming a changestream.
+#. ``$changeStream`` stage for ``ChangeStream`` started with ``startAfter`` against a server ``>=4.1.1`` that has not received any results yet MUST include a ``startAfter`` option and MUST NOT include a ``resumeAfter`` option when resuming a changestream.
```

Comment:
I realize you're just being consistent with previous usage, but can we write "change stream" instead of "changestream"? "ChangeStream" is one thing, since it maps to the class name in the spec, but "changestream" does not read well. Feel free to make a global replacement in this file if that's easier.

Reply:
I have been super confused by this usage myself. Why don't we simply use the class name `ChangeStream` everywhere in this document?